CHANGELOG.md

```diff
@@ -7,228 +7,50 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+- [#11915](https://github.com/MetaMask/metamask-extension/pull/11915): Improved time to see Swaps quotes
+- [#11802](https://github.com/MetaMask/metamask-extension/pull/11802): Replace hardcoded sent ether label on confirmation screen
+- [#11982](https://github.com/MetaMask/metamask-extension/pull/11982): Sort addresses within letter groups in address book / contacts alphabetically
+- [#12000](https://github.com/MetaMask/metamask-extension/pull/12000): Autofilling new issue link with url and domain (Phishing detect page)
+- [#11964](https://github.com/MetaMask/metamask-extension/pull/11964): Disabled spell-check on ens input
+- [#12013](https://github.com/MetaMask/metamask-extension/pull/12013): Improving warning text for eth_sign transactions
+- [#11945](https://github.com/MetaMask/metamask-extension/pull/11945): Send user to account page upon adding a custom network
+- [#11895](https://github.com/MetaMask/metamask-extension/pull/11895): Update header on Send page from `Add Recipient` -> `Send to`
+- [#11850](https://github.com/MetaMask/metamask-extension/pull/11850): Use "Secret Recovery Phrase" text throughout the onboarding flow
+- [#11795](https://github.com/MetaMask/metamask-extension/pull/11795): Autofocus the unit field in send screen
+- [#11889](https://github.com/MetaMask/metamask-extension/pull/11889): Update gas controls copy
+- [#11805](https://github.com/MetaMask/metamask-extension/pull/11805): Update tooltip text for gas fees
+- [#11796](https://github.com/MetaMask/metamask-extension/pull/11796): Update Max Fee label in the edit gas display
+- [#11845](https://github.com/MetaMask/metamask-extension/pull/11845): Wrapping and unwrapping native currencies (e.g. ETH -> WETH) through Swaps now uses the wrapping contracts directly
+- [#12056](https://github.com/MetaMask/metamask-extension/pull/12056): Updated labels and behaviour of duplicate cancel buttons on the send screen when editing a transaction
+
+### Fixed
+- [#12140](https://github.com/MetaMask/metamask-extension/pull/12140): Fixed max priority fee field so that there are not errors when attempting to enter more than 15 significant digits
+- [#12169](https://github.com/MetaMask/metamask-extension/pull/12169): Fixed erroneous ERC-20 Token amount on the confirmation screen
+- [#12074](https://github.com/MetaMask/metamask-extension/pull/12074): Fixed the addition of a duplicate, zero-balance account after importing a wallet with seed phrase
+- [#11944](https://github.com/MetaMask/metamask-extension/pull/11944): Fixed import account page layout
+- [#11967](https://github.com/MetaMask/metamask-extension/pull/11967): Fixed contact settings layout issues
+- [#12122](https://github.com/MetaMask/metamask-extension/pull/12122): Fixed stuck speedup transactions caused by prior failed speed up attempts
+- [#12098](https://github.com/MetaMask/metamask-extension/pull/12098): Fixed scrolling input issue in advanced gas options
```

Comment:
This seems more like a UX improvement :thinking: 

I don't think it's correct to describe this as fixing a scrolling issue. This PR _adds_ scrolling. Suggested rewording:
```suggestion
- [#12098](https://github.com/MetaMask/metamask-extension/pull/12098): Scroll down to show all fields when "Advanced options" is toggled in the advanced gas options
```

Reply:
updated